src/com/projectkorra/projectkorra/hooks/WorldGuardFlag.java

```diff
@@ -12,8 +11,8 @@ public static void registerBendingWorldGuardFlag() {
 		final FlagRegistry registry = WorldGuard.getInstance().getFlagRegistry();
 		try {
 			registry.register(new StateFlag("bending", false));
-		} catch (final FlagConflictException e) {
-			ProjectKorra.log.severe("unable to register bending WorldGuard Flag");
+		} catch (final Exception e) {
```

Comment:
You may want to include the Exception in the log message for debugging purposes. 
`"Unable to register bending WorldGuard flag: " + e`